packages/server/src/sdk/app/tables/utils.ts

```diff
@@ -9,3 +9,7 @@ export function isExternal(opts: { table?: Table; tableId?: string }): boolean {
   }
   return false
 }
+
+export function isTable(table: any): table is Table {
+  return table.type === "table"
```

Comment:
Its probably also worth checking the ID to see if it is in the form of a table ID - `isTableId` (I now realise these functions break the rule of capitalised `ID`

Reply:
I feel like if we detect `type == "table"` and _not_ a table ID, that's an error no? We should probably throw?

Reply:
Likely yes - there is cases where that could happen - we probably could add that here, the core to my point though is that you're better to look at the ID of the table than the `type` field, generally checking the ID is the preferred way to confirm what type a document/object is.

Reply:
Ahh I see. Done! Also changed isView to use isViewId.